pom.xml

```diff
@@ -485,6 +485,7 @@
                 <executions>
                     <execution>
                         <id>sort-imports</id>
+                        <phase>process-sources</phase>
```

Comment:
@fabiobrz this was handy, why did you remove it?

Reply:
I actually added it :slightly_smiling_face: